js/menu.js

```diff
@@ -133,6 +133,7 @@ function displayErr(input, msg) {
 
 function validateForm(input, msg) {
   //let x = document.forms["myForm"]["fname"].value;
```

Comment:
- [ ] (OPTIONAL) Kindly remove all commented code in order to keep your work clean 👍